app/models/record.server.ts

```diff
@@ -82,3 +82,15 @@ export async function doesRecordExist(data: Pick<Record, 'username' | 'type' | '
 export async function deleteRecordById(id: Record['id']) {
   return prisma.record.delete({ where: { id } });
 }
+
+export function renewRecordById(id: Record['id']) {
```

Comment:
Similar to one of my previous comments: could you change it to `renewDnsRecordById`?